src/applications/facility-locator/components/search-results/LocationInfoBlock.jsx

```diff
@@ -25,9 +25,15 @@ const LocationInfoBlock = ({ location }) => {
         </span>
       ) : (
         <span>
-          <Link to={`facility/${location.id}`}>
-            <h2 className="vads-u-font-size--h5">{name}</h2>
-          </Link>
+          {isVADomain(website) && process.env.BUILDTYPE === 'vagovstaging' ? (
```

Comment:
AFAIK this is a browser rendered component so `process.env` is not available in this context. This [article](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/platform/tools/feature-flags/) has info on how to do feature toggles including how to use [`isProduction()`](https://github.com/department-of-veterans-affairs/vets-website/blob/10380007a3ec77047be10e4fb2a42d343c22c5d3/src/platform/utilities/environment/index.js)

LMK if you need an overview of how the website and its build processes work. I'd also expect this to be caught in the peer review- don't approve PRs if you don't understand what they are doing.

Reply:
@rianfowler thanks for your feedback and the info you shared. I just updated the conditional to use isStaging(). darkbushido was mainly helping in the regular expression logic